can-view-model_test.js

```diff
@@ -2,6 +2,7 @@ var QUnit = require('steal-qunit');
 var viewModel = require('can-view-model');
 var SimpleMap = require('can-simple-map');
 var types = require('can-types');
+var $ = require("can-jquery");
```

Comment:
Hm, I wonder if instead of bringing in a dependency just for this test, the test should just create its own array-like object.